druid-shell/src/platform/x11/window.rs

```diff
@@ -593,19 +537,14 @@ impl Window {
             button,
             wheel_delta: Vec2::ZERO,
         };
-        match self.handler.try_borrow_mut() {
-            Ok(mut handler) => {
-                handler.mouse_up(&mouse_event);
-                Ok(())
-            }
-            Err(err) => Err(Error::BorrowError(format!(
-                "Window::handle_button_release handle: {}",
-                err
-            ))),
-        }
+        self.handler
+            .try_borrow_mut()
+            .context("Window::handle_button_release")?
```

Comment:
Lets keep the variable name in the context message. (`handler` instead of `handle` though)